docs/InstrumentationVerification/InstrumentationVerification.md

```diff
@@ -30,7 +30,7 @@ To run a verification, simply:
 2. Run or restart the application we are instrumenting. Observe that as we instrument the app, more and more files will appear under [Datadog Logs Folder]/InstrumentationVerification/{ProcessName}_{ProcessID}_{ProcessCreationTime}
```

Comment:
We can not delete step 1, because it is on by default, right?

Reply:
Fix in d6e34e581c70cc2271806c12db2c3d4f2caf78f1